PROJECTS.md

```diff
@@ -50,3 +50,10 @@ before contributing.
 - Great big example application (org)
     - [Great-big-example-application](https://github.com/dancancro/great-big-angular2-example) (project)
     - [Contributing Guide](https://github.com/dancancro/great-big-example-application/projects/1)
+- Public Lab (org)
+    - [Plots2](https://github.com/publiclab/plots2) (org)
```

Comment:
Could you change this to "(project)" instead of "(org)"?
(Public Lab is the org)

Reply:
@shubheksha thank for the review. I have updated the PR 👍